front/style/layout/_avatar.scss

```diff
@@ -43,6 +43,11 @@
     }
 
     &--style-02 {
+        background-color: $pink;
```

Comment:
@enzozccr, do we really want to change this? (it changes the style of avatars on CoTerr pages)

Reply:
ah, need to create a style-03 instead